src/app/server/Server.h

```diff
@@ -203,22 +205,23 @@ class Server
     public:
         ServerFabricDelegate() {}
 
-        CHIP_ERROR Init(SessionManager * sessionManager)
+        CHIP_ERROR Init(Server * server)
         {
-            VerifyOrReturnError(sessionManager != nullptr, CHIP_ERROR_INVALID_ARGUMENT);
+            VerifyOrReturnError(server != nullptr, CHIP_ERROR_INVALID_ARGUMENT);
 
-            mSessionManager = sessionManager;
+            mServer = server;
             return CHIP_NO_ERROR;
         };
 
         void OnFabricDeletedFromStorage(CompressedFabricId compressedId, FabricIndex fabricIndex) override
         {
             (void) compressedId;
-            if (mSessionManager != nullptr)
+            auto * sessionManager = &(mServer->GetSecureSessionManager());
```

Comment:
The reference here ensures that the pointer is not null. Why not use the reference directly? Should `GetSecureSessionManager()` return an pointer instead?

Reply:
`mServer` cannot be null based on Init()